docs/src/guide/processes.rst

```diff
@@ -290,7 +290,7 @@ A client which wants to connect to a domain socket will use::
 
     void uv_pipe_connect(uv_connect_t *req, uv_pipe_t *handle, const char *name, uv_connect_cb cb);
 
-where ``name`` will be ``echo.sock`` or similar.
+where ``name`` will be ``echo.sock`` or similar. ``name`` needs to point to a valid file on Unix systems, and needs the ``\\?\pipe\name`` format on Windows.
```

Comment:
How about "(...) ``echo.sock`` or similar. On Unix ``name`` (and stuff about unix). On Windows (and stuff about windows)"?